consutil/lib.py

```diff
@@ -326,15 +329,23 @@ def _parse_processes_info(output):
 
     @staticmethod
     def run_command(cmd, abort=True):
```

Comment:
>run_command [](http://example.com/codeflow?start=8&length=11)

Is it possible to implement this function by calling `run_command_pipe`? If yes, we can reuse some code.